src/Application/DegreeProgramBulkUpdater.php

```diff
@@ -19,10 +20,21 @@ public function __construct(
     }
 
     /**
-     * Degree program values are calculated based on the degree term.
-     * So if the term was changed, we have to update all related degree programs.
+     * Degree program slug is calculated based on the degree term meta abbreviation value.
+     * So if the abbreviation value was changed, we have to update all related degree programs.
+     *
+     * @wp-hook updated_term_meta
      */
-    public function whenDegreeTermUpdated(int $termId): void
+    public function whenTermMetaUpdated(int $metaId, int $objectId, string $metaKey): void
+    {
+        if ($metaKey !== Degree::ABBREVIATION) {
+            return;
+        }
+
+        $this->updateDegreeProgramsByTerm($objectId);
+    }
+
+    private function updateDegreeProgramsByTerm(int $termId): void
     {
```

Comment:
Does it make sense to merge the methods?
```suggestion
    public function whenTermMetaUpdated(int $metaId, int $termId, string $metaKey): void
    {
        if ($metaKey !== Degree::ABBREVIATION) {
            return;
        }
```

Reply:
Thanks for suggestion. In the current implementation, this can be done since we don't use the `updateDegreeProgramsByTerm` method anywhere else. However, I personally think it would be better to separate them, as the methods serve different purposes.